### ISSUE TITLE
Missing D?

### ISSUE THREAD

**Issue by boaks** [eclipse/dash-licenses#73]
The exmple

```sh
$ mvn org.eclipse.dash:license-tool-plugin:license-check -dash.iplab.token=<token> -dash.projectId=technology.dash
```

seems to miss `D`s, should be:

```sh
$ mvn org.eclipse.dash:license-tool-plugin:license-check -Ddash.iplab.token=<token> -Ddash.projectId=technology.dash
```

**Comment by waynebeaton** [comment]
Oops.

**Comment by waynebeaton** [comment]
Fixed in feaf61b423958e369a9b9ce83a12b6b1ec396e17
